src/integrationTest/groovy/wooga/gradle/fastlane/IntegrationSpec.groovy

```diff
@@ -26,124 +26,24 @@ import org.junit.contrib.java.lang.system.EnvironmentVariables
 import org.junit.contrib.java.lang.system.ProvideSystemProperty
 import spock.lang.Requires
 
-@Requires({ os.macOs })
-class IntegrationSpec extends nebula.test.IntegrationSpec {
+//@Requires({ os.macOs })
```

Comment:
This is to test the properties now against windows?

Reply:
Yes, that's correct.